SparkPost/2web2ui
# SA-1153 Link Watchlist Page to Incidents Page with Filter

### What Changed
 - Updated link on resources on the Watchlist page
 - Allow search value to come from URL on incidents page and update URL on blur
 - Fill search box with the resource from the URL

### How To Test
 - Click on an item in the watchlist, ensure this goes to incidents page, filtered by that resource
 - Type in the search box, click outside (blur) and ensure URL is updated with the new search value
